File: web3rpc/rpc-specs/code-samples/python/klay/transaction/getTransactionBySenderTxHash.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

transactionHash = "0x18fe9e1007da7d20aad77778557fb8acc58c80054daba65124c8c843aadd3478"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_transaction_by_sender_tx_hash(transactionHash)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/transaction/getTransactionBySenderTxHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const transactionHash = "0x21b2919b89278ca786226f10edbaadced7381dbd73df546a4823547aaebffa58"

    provider.klay.getTransactionBySenderTxHash(transactionHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()